hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -21,14 +21,20 @@ dependencies {
     api("com.typesafe:config:1.4.0")
     api("com.google.guava:guava:29.0-jre")
     api("com.graphql-java-kickstart:graphql-java-servlet:9.1.0")
```

Comment:
1.32.1 is vulnerable?

Reply:
It's actually netty `4.1.51.Final` that's vulnerable (https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439)- which is used both by 1.32.1 and 1.33. I upgraded grpc hoping that would update netty, saw it didn't, so then explicitly updated netty at the bottom.